internal/tools/go.mod

```diff
@@ -5,9 +5,9 @@ go 1.17
 require (
 	github.com/frapposelli/wwhrd v0.4.0
 	github.com/go-enry/go-license-detector/v4 v4.3.0
-	github.com/golangci/golangci-lint v1.45.2
+	github.com/golangci/golangci-lint v1.47.1
```

Comment:
Worth trying to bump to v1.47.2 and see if it fixes the process being killed on CircleCI